### DISCUSSION TITLE
label and relationship config not working.

### DISCUSSION THREAD

**Discussion by gustavoK47** [neo4j-contrib/neovis.js#266] [Q&A]
Good evening, 
I am trying to integrate neovis into my web application but I am having some issues in creating the test application that uses the Russian trolls db. Basically I am preparing my config object but, it is not getting the label and relationship properties from my object. 
I tried putting a breakpoint in the buildNodeVisObject(n) function but when I load my graph the passed n.labels[0] is never the label of the node I want it to edit the properties of...
Am I doing something wrong?

Thanks,
Gustavo.

**Comment by gustavoK47** [comment]
`                var config={
                    container_id: "viz",
                    server_url: "bolt://ip_address:port",
                    server_user: "user",
                    server_password: "password",
                    labels: {
                        "Troll": {
                            caption: "user_key",
                            size: "pageRank",
                            community: "community"
                        }
                    },
                    realtionships: {
                        "RETWEETS": {
                            caption: false,
                            thickness: "count"
                        }
                    },
                    initial_cypher: "cypher query"
                }`

this is my config object for reference.

**Comment by gmpmovies** [comment]
I ran into the same issue, try swapping out 
`labels: { "Troll": { caption: "user_key", size: "pageRank", community: "community" } },` for `labels: { "User": { caption: "user_key", size: "pageRank", community: "community" } },`
